src/components/ReleaseMonitor/ReleaseListHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import { createTableHeaders } from '../../shared/components/table/utils';
+
+export const releaseTableColumnClasses = {
+  name: 'pf-m-width-20 ',
+  status: 'pf-m-width-10',
+  completionTime: 'pf-m-width-10',
+  component: 'pf-m-width-20',
+  application: 'pf-m-width-10',
+  releasePlan: 'pf-m-width-20',
+  namespace: 'pf-m-width-10 ',
+};
+
+export const enum SortableHeaders {
+  name,
+  created,
+}
+
+const releaseColumns = [
+  { title: 'Release Name', className: releaseTableColumnClasses.name },
```

Comment:
you defined the "SortableHeaders", but you didn't add the `sortable: true`  prop to any of the columns here . so the "sorting" logic will never work :)

Reply:
if you don't really need having the sorting logic to any column, that we could simply remove the code related to it

Reply:
Removed, thanks.